web/src/main/webapp/WEB-INF/tags/administrationTemplate.tag

```diff
@@ -0,0 +1,34 @@
+<%-- 
+    Document   : administrationTemplate
+    Created on : 15.12.2018, 13:25:19
+    Author     : xtrnkal
+--%>
+
+<%@tag description="put the tag description here" pageEncoding="UTF-8" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="my" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ attribute name="body_area" fragment="true" %>
+<%@ attribute name="subtitle" required="false" %>
+
+<my:pagetemplate title="Administration">
+    <jsp:attribute name="body">
+        <div class="btn-group btn-group-justified">
+            <a href="/web/show/" class="btn btn-primary">Shows</a>
```

Comment:
Dej tam `${pageContext.request.contextPath}`, at si muzeme byt jisti, ze se to nemuze rozbit